README.md

```diff
@@ -57,9 +64,9 @@ const createSlackEventAdapter = require('@slack/events-api').createSlackEventAda
 const slackEvents = createSlackEventAdapter(process.env.SLACK_VERIFICATION_TOKEN);
 const port = process.env.PORT || 3000;
 
-// Attach listeners to events by Slack Event. See: https://api.slack.com/events/api
-slackEvents.on('message.im', (event)=> {
-  console.log(`Received a DM event: user ${event.user} in channel ${event.channel} says ${event.text}`);
+// Attach listeners to events by Slack Event "type". See: https://api.slack.com/events/message.im
```

Comment:
im not sure actually. there's two very similar things that i'm trying to differentiate. the first is the Event Name, which is what you choose to subscribe to in you app configuration (and in this case its `message.im`). the second is the Slack Event "type" as i chose to call it, which is the value for the `type` property of in the payload (and in this case its `message`). any suggestions on how to best describe this here?